.github/workflows/.windows.yml

```diff
@@ -92,7 +92,6 @@ jobs:
           & docker build `
             --build-arg WINDOWS_BASE_IMAGE `
             --build-arg WINDOWS_BASE_IMAGE_TAG `
-            --build-arg GO_VERSION `
```

Comment:
Was this intended?

Reply:
Possibly related to that as well. That said, I wonder if we need to pass it, given that we update the `GO_VERSION` inside the Dockerfile as well 🤔 But probably best for separate.